gef.py

```diff
@@ -9962,12 +9963,15 @@ def tmux_setup(self) -> None:
         forcing the context to be redirected there."""
         tmux = which("tmux")
         ok("tmux session found, splitting window...")
-        old_ptses = set(os.listdir("/dev/pts"))
-        gdb.execute(f"! {tmux} split-window -h 'clear ; cat'")
+        
+        pane_info = subprocess.check_output([tmux, "splitw", "-h", '-F#{session_name}:#{window_index}.#{pane_index}-#{pane_tty}', "-P"]).decode().strip().split("-")
+        pane = pane_info[0]
+        pty = pane_info[1]
```

Comment:
Minor edit, might as well simplify like this since `pane_info` is only used once
```suggestion
        pane, pty = subprocess.check_output([tmux, "splitw", "-h", '-F#{session_name}:#{window_index}.#{pane_index}-#{pane_tty}', "-P"]).decode().strip().split("-")
```